File: src/targets/python/python3/fixtures/multipart-form-data-no-params.py
import http.client

conn = http.client.HTTPConnection("mockbin.com")

payload = ""

headers = { 'Content-Type': "multipart/form-data" }

conn.request("POST", "/har", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: src/targets/node/unirest/fixtures/multipart-form-data-no-params.js
const unirest = require('unirest');

const req = unirest('POST', 'http://mockbin.com/har');

req.headers({
  'Content-Type': 'multipart/form-data'
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});